src/components/sharing/constants.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * These are the keys to each permission-lister response object and correspond
+ * to each type of resource that can be shared in Sonora
+ */
+export const TYPE = {
+    DATA: "paths",
+    APPS: "apps",
+    ANALYSES: "analyses",
+    TOOLS: "tools",
+};
```

Comment:
I separated these out into another file because this is the only part of the sharing utils namespace that the Bag component was using, but since it was in that file, it brought in the whole thing. Since Bag is also used by CyVerseAppBar, this saved a few kB.